src/app/docs/back-to-top/back-to-top-docs.component.html

```diff
@@ -4,10 +4,39 @@
   pageTitle="Back to top"
 >
   <sky-docs-demo-page-summary>
-    Sample description.
+    Adds a back to top button to the page when the element is scrolled out of view.
   </sky-docs-demo-page-summary>
 
-  <sky-docs-design-guidelines>
+  <sky-docs-demo>
+    <div class="sky-text-info sky-margin-stacked-content-section">We are creating a back to top component demo to fit in this section. To view the existing demo in the meantime, select the Development tab.</div>
+  </sky-docs-demo>
+
```

Comment:
I don't see JSDoc comments in this PR, but when I open the build link to preview this page, I see a directives section with an entry under a `SkyBackToTopDirective` header. Where is that text? I'm adding my feedback for that text here since I can't find it in the PR... The current text is: "Adds a back to top button to the page when the element is scrolled out of view. When users click on the back to top button, the browser will scroll back to the element this directive was placed on."

What do you think about tightening this up to focus on implementation details? Since we should have a Design tab at some point with behavior/uses info, I think we can put the longer explanation in the overview for now. I'm thinking of something like: "Associates a button with an element on the page and displays a button to return to that element after users scroll away from it."


Also, do we need a section for the component? I don't see the `sky-back-to-top` selector used in any of the code samples, but just want to make sure.

Reply:
That jsdoc comment is in the actual `back-to-top.directive.ts` file. I've updated it to reflect your comment.